.storybook/preview.tsx

```diff
@@ -32,7 +33,9 @@ const AppDecorator: Exclude<Preview["decorators"], undefined>[0] = (
 
   return (
     <L10nProvider bundleSources={l10nBundles}>
-      <ReactAriaI18nProvider locale="en">{storyFn()}</ReactAriaI18nProvider>
+      <SessionProvider session={null}>
```

Comment:
Just setting the session to `null` should do here. In places where we mock session and/or user data, we do that in the respective Stories — like here on the Dashboard: https://github.com/mozilla/blurts-server/blob/MNTOR-2324/storybook-session-provider/src/app/(proper_react)/redesign/(authenticated)/user/dashboard/Dashboard.stories.tsx#L152.